datadog_checks_dev/datadog_checks/dev/tooling/commands/release/changelog.py

```diff
@@ -101,81 +87,46 @@ def changelog(
         write_result(dry_run, changelog_path, ''.join(old), num_changes=1)
         return
 
-    user_config = ctx.obj
-    entries = defaultdict(list)
-    generated_changelogs = 0
-    for pr_num in pr_numbers:
-        try:
-            payload = get_pr(pr_num, user_config, org=organization)
-        except Exception as e:
-            echo_failure(f'Unable to fetch info for PR #{pr_num}: {e}')
-            continue
-
-        changelog_labels = get_changelog_types(payload)
-
-        if not changelog_labels:
-            abort(f'No valid changelog labels found attached to PR #{pr_num}, please add one!')
-        elif len(changelog_labels) > 1:
-            abort(f'Multiple changelog labels found attached to PR #{pr_num}, please only use one!')
-
-        changelog_type = changelog_labels[0]
-        if changelog_type == CHANGELOG_TYPE_NONE:
-            if not quiet:
-                # No changelog entry for this PR
-                echo_info(f'Skipping PR #{pr_num} from changelog due to label')
-            continue
-
-        generated_changelogs += 1
+    # find the first header below the Unreleased section
```

Comment:
Almost related: should we add the unreleased section to the templates? For instance [here](https://github.com/DataDog/integrations-core/blob/90da5f6801797b76af84713911ed39ae57634fef/datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/check/%7Bcheck_name%7D/CHANGELOG.md#L5-L4)

Reply:
done!